geniuspaste/src/geniuspaste.c

```diff
@@ -94,6 +94,54 @@ PLUGIN_SET_TRANSLATABLE_INFO(LOCALEDIR, GETTEXT_PACKAGE, PLUGIN_NAME,
                              PLUGIN_VERSION, "Enrico Trotta <enrico.trt@gmail.com>")
 
 
+/* Windows compatibility for resolving system plugin data path.
+ * This only works if the plugin's data is installed in the same tree as the
+ * plugin DLL, which might not be true (as the DLL has to be in Geany's libdir
+ * but the plugin's data might be anywhere in theory) */
```

Comment:
So, what do we want to do?
I'm not sure how relevant supporting any possible installation scenario on Windows really is. I pretty much assume most users just install Geany. Maybe there are two or three developers who actually fight with developing on Windows...

Regarding `libgeany-plugins`: sounds interesting and would ease a lot of things probably. Something we should consider for 1.28.